src/services/notification.ts

```
@@ -217,6 +216,17 @@ export class NotificationService {
 		});
 	}
 
+	static async sendVendorNotification(
+		vendorId: string,
+		notificationData: Omit<CreateNotificationData, "userId">,
+	): Promise<void> {
+		await this.create({
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)